classifiers/reference_complexity/maximum_sentence_complexity/code_snippet_refinery.md

```diff
@@ -0,0 +1,28 @@
+```python
+import textstat
+
+ATTRIBUTE: str = "text" # only text attributes
+TARGET_LANGUAGE: str = "en" # iso codes
+
+def maximum_sentence_complexity(record):
+    complexities = [textstat.flesch_reading_ease(sent.text) for sent in record[ATTRIBUTE].sents] 
+    return get_mapping_complexity(min(complexities))
+
+def get_mapping_complexity(score):
+    if score < 30:
+        return "very difficult"
+    if score < 50:
+        return "difficult"
+    if score < 60:
+        return "fairly difficult"
+    if score < 70:
+        return "standard"
+    if score < 80:
+        return "fairly easy"
+    if score < 90:
+        return "easy"        
+    return "very easy"
+
+if TARGET_LANGUAGE is not None:
+    textstat.set_lang(TARGET_LANGUAGE)
```

Comment:
suggest to move this up under the definition of `TARGET_LANGUAGE`. makes it easier to read the code

Reply:
Sure, I've moved it!